crates/usvg-parser/src/filter.rs

```diff
@@ -99,31 +96,30 @@ pub(crate) fn convert(
             } => create_base_filter_func(
                 convert_drop_shadow_function(node, color, dx, dy, std_dev, state),
                 &mut filters,
-                cache,
             ),
             svgtypes::FilterValue::Brightness(amount) => {
-                create_base_filter_func(convert_brightness_function(amount), &mut filters, cache)
+                create_base_filter_func(convert_brightness_function(amount), &mut filters)
```

Comment:
Why no cache?

Reply:
Honestly, I didn't question it. I just pulled the same changes that resvg made from 0.36-0.37 (all of the changes that were compatible anyways). Is that something you think I should dig deeper into?

Reply:
No, just wanted to understand anything that we did and I was not sure what was what.